.github/workflows/tagging.yml

```diff
@@ -19,17 +19,17 @@ jobs:
           echo $commitmsg
       - name: get tag version
         run: |
-          echo ::set-env name=tagversion::$( sed -e 's#.*: \(\)#\1#' <<< ${commitmsg} )
-          echo $tagversion
+          echo ::set-env name=tagversion::$( egrep -o '([0-9]+\.){2}(\*|[0-9]+)(-.*)?' <<< ${commitmsg} )
+          echo v$tagversion
```

Comment:
what happens if you add the v? will it append it twice?

Reply:
looks like it only pulls the version

Reply:
@infoxicator  as Jonny mentioned it would only pull the version therefore we can avoid any duplication of the `v`